docs/node-operators/querying-data.mdx

```diff
@@ -11,19 +15,23 @@ import Subhead from "@site/src/components/common/Subhead";
 Data querying for node operators.
 </Subhead>
 
-Some node operators don't wish to participate in block production or snark work creation, but just want to sit on the network and passively aggregate information for querying or submit signed transactions. These nodes don't require an account keypair themselves to run successfully.
+Some node operators don't wish to participate in block production or SNARK work creation, but just want to sit on the network and passively aggregate information for querying or submit signed transactions. These nodes don't require an account key pair themselves to run successfully.
 
 ## Preparation
 
-In addition to [installing the Mina daemon on your machine](/node-operators/getting-started), you may also want to have some system set up to archive historical information on your machine. To do so you can setup an [archive node](/node-operators/archive-node) and/or archive via [other redundant means](/node-operators/archive-redundancy).
+In addition to [installing the Mina daemon on your machine](/node-operators/getting-started), you may also want to have some system set up to archive historical information on your machine. To do so you can setup an [archive node](/node-operators/archive-node), archive by using [other redundant means](/node-operators/archive-redundancy), or both.
 
 ## Running a node
 
-The best way to learn how to run a node is to follow the instructions on the [Connecting page](connecting-to-the-network) of the docs.
+The best way to learn how to run a node is to follow the [Connect to the Mina Network](connecting-to-the-network) instructions.
 
-However, you should remove `MINA_PRIVKEY_PASS` and `-block-producer-key ...` flag as we do not wish this node to start as a block producer.
+To prevent this node from starting as a block producer, remove the `MINA_PRIVKEY_PASS` and `-block-producer-key ...` flags.
```

Comment:
"MINA_PRIVKEY_PASS" is an environment variable rather than a flag. So it should read something like "...remove the `MINA_PRIVKEY_PASS` variable, and `-block-producer-key ...` flag."